crates/rust-analyzer/src/config.rs

```diff
@@ -4088,4 +4096,14 @@ mod tests {
             matches!(config.flycheck(None), FlycheckConfig::CargoCommand { options, .. } if options.target_dir == Some(Utf8PathBuf::from("other_folder")))
         );
     }
+
+    #[test]
+    fn test_default_comments_enabled() {
```

Comment:
Please remove this. We don't test the default config in other places, it's also a wrong idea to test it.